Sources/AcmeSwift/Models/AcmeError.swift

```diff
@@ -19,8 +19,6 @@ public enum AcmeError: Error, Sendable {
     
     case jwsEncodeError(String)
     
-    case invalidKeyError(String)
-    
```

Comment:
This is no longer needed as we are no longer hopping through JWTKit for the key parameters.